web/src/app/admin/admin-outgoing-logs/OutgoingLogsList.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect } from 'react'
+import { Box } from '@mui/system'
+import { DateTime } from 'luxon'
+import { DebugMessage } from '../../../schema'
+import OutgoingLogCard from './OutgoingLogCard'
+import { useFuse } from './hooks'
+import { useURLParam } from '../../actions'
+import { Typography, Button } from '@mui/material'
+
+export const LOAD_AMOUNT = 50
+
+interface Props {
+  debugMessages?: DebugMessage[]
+  selectedLog: DebugMessage | null
+  onSelect: (debugMessage: DebugMessage) => void
+}
+
+export default function OutgoingLogsList(props: Props): JSX.Element {
+  const { debugMessages = [], selectedLog, onSelect } = props
+
+  const [searchTerm] = useURLParam('search', '')
+  const [start] = useURLParam('start', '')
+  const [end] = useURLParam('end', '')
+  const [limit, setLimit] = useURLParam<string>('limit', '1')
```

Comment:
Same as elsewhere, remove the `_limit` and pass an integer as the default value to have the type always be an integer.

Secondly, I think this might make sense to be the total count -- as a URL parameter, I think it's going to make more sense to a user to see `limit=100` meaning 100 results than `limit=2` meaning 100 results.

We should change it elsewhere to keep consistent

```suggestion
  const [limit, setLimit] = useURLParam<string>('limit', LOAD_AMOUNT)
```